account-kit/rn-signer/account-kit-react-native-signer.podspec

```diff
@@ -11,12 +11,15 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => min_ios_version_supported }
+  s.platforms    = { :ios => "17.0" }
   s.source       = { :git => "https://github.com/alchemyplatform/aa-sdk.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm,cpp,swift}"
   s.private_header_files = "ios/**/*.h"
 
+  # spm_dependency(s, url: "https://github.com/tkhq/swift-sdk", requirement: {kind: 'upToNextMajorVersion', minimumVersion: '1.2.0'}, products: ["TurnkeySDK"])
```

Comment:
removing this because I can't get a build working using an `spm_dependency` because it requires the use of `use_frameworks!` with dynamic linking which breaks `react-native-mmkv`